roles/shiftstack/templates/shiftstackclient_pod.yml.j2

```diff
@@ -28,6 +28,8 @@ spec:
     - mountPath: /home/cloud-admin/.original-config/cert/
       name: openstack-cert
       readOnly: true
+    - name: artifacts-volume
+      mountPath: {{ cifmw_shiftstack_artifacts_dir }}
```

Comment:
Please mount the PVC on the `installation` directory instead of the `artifacts` directory.